Tools/canard_dsdlc/templates/Makefile.em

```diff
@@ -0,0 +1,39 @@
+@{from canard_dsdlc_helpers import *}@
+INC=-I../include
+INC+=-I@(get_canard_inc())
+INC+=-I.
+CXX=g++
+CC=gcc
+CFLAGS=-Wall -Wextra -Werror -m32 -std=c99 -DCANARD_DSDLC_TEST_BUILD $(INC)
```

Comment:
why not C11 ?

Reply:
If it doesn't matter the greater is the better. 
Without joke that would be better to have the same norm everywhere and don't make static analyzer goes crazy in a near future